step.yml

```diff
@@ -261,7 +272,7 @@ inputs:
       category: Debug
       title: "Run xcodebuild test only"
       description: |-
-        If `single_build` is set to false, the step runs `xcodebuild OPTIONS build OPTIONS` before the test 
+        If `single_build` is set to false, the step runs `xcodebuild OPTIONS build OPTIONS` before the test
```

Comment:
step > Step

Reply:
If `single_build` is set to false, the step runs `xcodebuild OPTIONS build OPTIONS` before the test. (period was missing)